packages/spor-react/src/loader/SkeletonText.tsx

```diff
@@ -1,7 +1,10 @@
-import { BoxProps, SkeletonText as ChakraSkeletonText } from "@chakra-ui/react";
+import { BoxProps, SkeletonText as ChakraSkeletonText, SkeletonProps } from "@chakra-ui/react";
 import React from "react";
 
-export type SkeletonTextProps = BoxProps;
+export type SkeletonTextProps = BoxProps & {
+  skeletonHeight?: SkeletonProps["height"]
+};
```

Comment:
Hm – kanskje man heller kan eksportere typen til Chakra direkte?

```tsx
import { SkeletonTextProps as ChakraSkeletonTextProps } from "@chakra-ui/react";

export type SkeletonTextProps = ChakraSkeletonTextProps;
```

Så enabler vi alle muligheter for SkeletonText?

Reply:
mye bedre. slipper dobbelt arbeid 👍